src/docs/releases/1.9.0.md

```diff
@@ -18,6 +18,13 @@ If you needed to enable indexing for other extensions like (`.docx`, or `.pptx`)
 
 Introducing a new "Azure AI Search" module, designed to empower you in the administration of Azure AI Search indices. When enabled with the "Search" module, it facilitates frontend full-text search capabilities through Azure AI Search. For more info read the [Azure AI Search](../reference/modules/AzureAISearch/README.md) docs.
 
+### Data Access
+
+Previousle, `yessql.db` file was used as a default SQLite database for newly created tenant. Now `OrchardCore.db` has been used instead, but you could configure the database name using `SqliteOptions.Database`.
+
+!!! note
+    For backward compatibility you could use `yessql.db` as database name in the `OrchardCore_Data_Sqlite` configuration. For more info read the [Data (OrchardCore.Data)](../reference/core/Data/README.md) docs.
```

Comment:
```suggestion
Previously, the `yessql.db` file was used as a default SQLite database for newly created tenants. Now `OrchardCore.db` is used instead, but you can configure the database name using `SqliteOptions.Database`.

!!! warning
    For backward compatibility, you should use `yessql.db` as the database name in the `OrchardCore_Data_Sqlite` configuration. For more info read the [Data (`OrchardCore.Data`) documentation](../reference/core/Data/README.md).
```

Reply:
This isn't fully addressed.

Reply:
Sorry something accidentally happen during the merge conflict last time